### ISSUE TITLE
Typography Audit Issue: Homepage Email Sign up section

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#18078]
## Status
[2025-02-12] See this [comment](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/18078#issuecomment-2110591904) regarding custom styling needs to be removed
**[2024-08-01] [Fran]** The work to remove the custom code in this [ticket](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/18111) has been completed, and I've confirmed the first three AC items. This now needs design review, so am leaving it in Ready and adding a comment to Jordan/Dave letting them know we'll pull into the upcoming sprint, if possible.
**[2024-05-16]** A [ticket was created to remove custom code](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/18111). After the custom code is removed and everything tested, it may correct any issues and at which time this ticket may be closed as completed.
**[2024-05-09]** Comment asking Amanda to review and provide determination on outcome.

## New Description

**[2025-02-12]** See this [comment](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/18078#issuecomment-2110591904) regarding custom styling needs to be removed

#### Original Description

From Phase 1 of the Typography audit, we found a ew issues on the homepage in the bottom Sign up for email section:

1. The H2 heading for "Sign up to get the latest VA updates is much larger in the PR than it currently is on Prod. 
![image (4)](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/9f11f896-6050-48f6-bc36-1ecebfcdc26b)
  - We are using a custom style, a `part` to set the size instead of just using the default DS sizes here. So if we were to remove the custom piece this should sort itself out.
  - However, this was a Design from Ryan Thurlwell and was approved by folks and was a big VA initiative. Can we update to the DS size here or would we need approval by folks?

2. The top margin on the Sign up button is slightly smaller on the PR. I am seeing something in the CSS that doesn't seem to be formatted correctly that I think might be causing this?
![image (5)](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/4a00c2f7-558e-40c8-88a3-4a5f996f9cce)

3. The the input field for the email sign up is larger on the PR
![Screenshot 2024-05-09 at 9 08 08 AM](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/eec415ec-8883-43ee-82a4-020418f23e9b)

## New Acceptance Criteria
- [ ]  Custom styling has been removed per this [comment ](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/18078#issuecomment-2110591904)below

## Original Acceptance Criteria
- [ ]  Custom code has been removed (see this [comment](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/18078#issuecomment-2110591904) below)
- [ ] Email sign up heading is the proper size
- [x] Sign up button has the proper top margin
- [x] Input field is the proper width
- [x] Requires design review

**Comment by FranECross** [comment]
@aklausmeier UX has looked at this and we would like you to review to let us know if it should match production exactly, or if we should defer to the new design system component/typography change and not fix/be concerned? Thanks in advance!

**Comment by laflannery** [comment]
I had mentioned that I thought DS standard styles would make the H2 somewhere inbetween current Prod and the Typography PR - and technically that was correct but not as drastic as I had thought
- The Typography PR is 32px
- Standard DS size is only 30px so not a very big difference. Screenshot with standard styles for elements:
![Screenshot 2024-05-09 at 3 22 43 PM](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/d0bb098a-eb16-4947-97aa-26c2ba6089f3)

**Comment by aklausmeier** [comment]
Let's align with design system, removing custom. @FranECross @laflannery

**Comment by randimays** [comment]
Tested this locally and noticed that all the issues called out in the description are fixed. The local version matches the prod version. But also acknowledging that there is custom styling on this section that we can use this ticket to remove 👍🏻

**Comment by FranECross** [comment]
@randimays [Ticket](Typography Audit: Remove Homepage Email Sign up section #18111) created and in Next Refinement. Thanks again! cc @aklausmeier @laflannery

**Comment by FranECross** [comment]
Closing per comments above.

**Comment by aklausmeier** [comment]
Can we make sure custom coding can be removed before closing? @randimays

**Comment by laflannery** [comment]
I think it also  might be worth noting that the RI still looks like the screenshot in the original description - so it seems like there are differences in what folks are seeing locally and what the RI shows

**Comment by randimays** [comment]
@aklausmeier Yes, we can remove the custom styling. I made some quick changes in the browser on production to simulate this, and this is what I'm gathering the final result should look like across breakpoints:

### 320px
<img width="318" alt="Screenshot 2024-05-14 at 10 54 21 AM" src="https://github.com/department-of-veterans-affairs/va.gov-cms/assets/19175324/9a6c65d1-632a-4d94-8707-1bed31d8c5bf">

### 480px
<img width="480" alt="Screenshot 2024-05-14 at 10 54 31 AM" src="https://github.com/department-of-veterans-affairs/va.gov-cms/assets/19175324/029b1210-5474-4057-a9eb-4b1e1b943ecd">

### 767px
<img width="766" alt="Screenshot 2024-05-14 at 10 54 39 AM" src="https://github.com/department-of-veterans-affairs/va.gov-cms/assets/19175324/975ffdb0-5287-4dfc-b063-889b0f0e699b">

### 768px
<img width="764" alt="Screenshot 2024-05-14 at 10 54 46 AM" src="https://github.com/department-of-veterans-affairs/va.gov-cms/assets/19175324/6b7a078c-dcdd-40ae-89ad-1cd388a81cb0">

### 1024px
<img width="1024" alt="Screenshot 2024-05-14 at 10 54 54 AM" src="https://github.com/department-of-veterans-affairs/va.gov-cms/assets/19175324/521320a7-3ff1-4a00-a4a1-8cbead6c4e70">

### 1400px
<img width="1399" alt="Screenshot 2024-05-14 at 10 55 05 AM" src="https://github.com/department-of-veterans-affairs/va.gov-cms/assets/19175324/d9927eab-6977-4d53-adb2-9274de3b47d2">

**Comment by aklausmeier** [comment]
looks good to me @randimays

**Comment by FranECross** [comment]
@thejordanwood @davidmpickett The work to remove the custom code in this [ticket](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/18111) has been completed, and I've confirmed the first three AC items. The last AC is for design review. Any objections to having us include this in the upcoming sprint? Thanks!

**Comment by laflannery** [comment]
@FranECross This ticket has not been completed, this will require an engineering PR to do the work

**Comment by jilladams** [comment]
Moving this into the 2023 homepage follow up epic, since it's the last ticket standing in the typography audit batch (in service of GHP migratio).

**Comment by randimays** [comment]
Pulled in for Sprint 1-25.

**Comment by randimays** [comment]
@davidmpickett This is ready for a design review in [staging](https://staging.va.gov/) please.

**Comment by davidmpickett** [comment]
@randimays I'm having trouble parsing what the before and after states are here. I'm not seeing any difference between Staging and Prod?

**Comment by randimays** [comment]
@davidmpickett Sorry about that - this ticket ended up being a little weird. We need to validate that the space above and below the button for the form and the font sizes are as we expect them to be. I just removed a couple of custom styles from those elements.

**Comment by randimays** [comment]
`part="form-header"` is part of the web component code coming from DST. We were targeting this as a selector for CSS previously, and now it has been removed. Screenshot below of the PR for this ticket where I removed the style:

<img width="1099" alt="Image" src="https://github.com/user-attachments/assets/5a19606b-a939-4484-835c-d2ec366200df" />

I'll go ahead and close this ticket as complete. Feel free to holler / re-open if you notice any weirdness.
